howtos/diffs/scheduled-sampling.diff

```diff
@@ -123,3 +122,22 @@ index 60b79b5..b1b8f18 100644
 +            metrics['accuracy'] * 100)
          decode_batch(optimizer.target, 5)
    return optimizer.target
+ 
+diff --git a/examples/seq2seq/train_test.py b/examples/seq2seq/train_test.py
+index b638d55..26fc7b8 100644
+--- a/examples/seq2seq/train_test.py
++++ b/examples/seq2seq/train_test.py
+@@ -101,7 +101,7 @@ class TrainTest(absltest.TestCase):
+       model = train.create_model()
+       optimizer = train.create_optimizer(model, 0.003)
+       optimizer, train_metrics = train.train_step(
+-          optimizer, batch, nn.make_rng())
```

Comment:
This doesn't look right to me. Have the arguments to `train_step` changed? I don't see that in the rest of the HOWTO diff above.

Reply:
OH SORRY MY BAD YOU'RE RIGHT.